internal/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -88,7 +88,7 @@ RUN mkdir -p ${HOME}/.ansible/tmp \
  && chown -R ${USER_UID}:0 ${HOME} \
  && chmod -R ug+rwx ${HOME}
```

Comment:
missing add the fragment for the fix.

Reply:
Are you asking me to change something here?

Reply:
@agaffney I think @camilamacedo86 is asking for the changelog fragment.

Here's the template: https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml

Reply:
Ah, I saw nothing about that in `CONTRIBUTING.md`, and I didn't see anything like this when I looked at some of the recently merged PRs after the above comment. This hardly seems worthy of a changelog entry, but I'll go ahead and create the fragment.

Reply:
Done